lib/lndclient/LndClient.ts

```diff
@@ -597,6 +596,38 @@ class LndClient extends SwapClient {
     }
   }
 
+  public lookupPayment = async (rHash: string) => {
+    const payments = await this.listPayments(true);
+    for (const payment of payments.getPaymentsList()) {
+      if (payment.getPaymentHash() === rHash) {
+        const paymentStatus = payment.getStatus();
+        if (payment.getStatus() === lndrpc.Payment.PaymentStatus.SUCCEEDED) {
```

Comment:
nit: would a switch statement here look better?

Reply:
Indeed it would.